### PR TITLE
Allow to run testnet-automation.sh directly from PRs

### PR DESCRIPTION
**Pull request by ryoqun**
#### Problem

While doing #9161, I noticed that I can't test testnet automation scripts with my changes in that PR.

#### Summary of Changes

I'm surprised testnet-automation.sh is really solid to this extent; just a small more wiring was needed to fix the problem.

_As a bonus, this paves the way to run preset of testnet runs from PRs directly and automatically._

Well, adequately streamlined, yet invoking bunch of `colo.sh`/`gcp.sh` and `net.sh` is a bit tedious for me just to run normal perf. tests. You know, engineers are lazy.. :p

### CODE REVIEW

File: system-test/testnet-automation.sh [hunk 1]
```diff
@@ -121,9 +124,17 @@ function launch_testnet() {
 
   execution_step "Starting bootstrap node and ${NUMBER_OF_VALIDATOR_NODES} validator nodes"
   if [[ -n $CHANNEL ]]; then
+    if [[ -z $NDEBUG ]]; then
+      maybeDebug=--debug
+    fi
+    # Our build machne is on Azure which supports AVX512 but colos don't support AVX512.
+    # So, turn off simd at all as a work around to correctly cross-compile
```
Comment (ryoqun) [inline system-test/testnet-automation.sh:131]: I need to create a tiny build-related PR against solana-reed-solomon to remove this work-around.

File: system-test/testnet-automation.sh [hunk 2]
```diff
@@ -121,9 +124,17 @@ function launch_testnet() {
 
   execution_step "Starting bootstrap node and ${NUMBER_OF_VALIDATOR_NODES} validator nodes"
   if [[ -n $CHANNEL ]]; then
+    if [[ -z $NDEBUG ]]; then
+      maybeDebug=--debug
+    fi
+    # Our build machne is on Azure which supports AVX512 but colos don't support AVX512.
```
Comment (ryoqun) [inline system-test/testnet-automation.sh:130]: err, `machine`.

**Comment by danpaul000** [comment]
@ryoqun I don't quite follow where the direct-from-PR trigger is here.  I think you are aware of the doc here for running test cases against a PR commit, yes?  https://docs.google.com/document/d/1h1nYR7uwCNxntkKCvbsx64rVPGhhcH1Ub47rVYYVFZw/edit#heading=h.fxcmhq1uji4u

So we can run against a PR already, but not directly triggered from github, still requires a few clicks in buildkite.

File: system-test/testnet-automation.sh (hunk 1 above)
Comment (ryoqun) [reply system-test/testnet-automation.sh:131]: @mvines I just created!: https://github.com/solana-labs/reed-solomon-erasure/pull/1

**Comment by ryoqun** [comment]
> @ryoqun I don't quite follow where the direct-from-PR trigger is here. I think you are aware of the doc here for running test cases against a PR commit, yes? https://docs.google.com/document/d/1h1nYR7uwCNxntkKCvbsx64rVPGhhcH1Ub47rVYYVFZw/edit#heading=h.fxcmhq1uji4u
> 
> So we can run against a PR already, but not directly triggered from github, still requires a few clicks in buildkite.

Cool!! So, we have that feature in place.. I created this pr while mistakenly thinking I cannot deploy PR changes to colo from wrong assumption/code reading... And, I've read that doc in the distant past and I've forgotten... Sorry for confusing by lack of understanding!

Anyway, I'll give it a shot!

**Comment by ryoqun** [comment]
(I'm trying to boot this aged pr for test-upgrade pr): the successful build https://buildkite.com/solana-labs/testnet-automation-colocation/builds/130#565ed5c7-b9f4-4842-807f-2165b16e3184

**Pull request closed without merging**